pkg/osvscanner/osvscanner.go

```diff
@@ -704,14 +704,21 @@ func filterPackageVulns(r reporter.Reporter, pkgVulns models.PackageVulns, confi
 				for _, id := range group.Aliases {
 					ignoredVulns[id] = struct{}{}
 				}
+
+				reason := ignoreLine.Reason
+
+				if reason != "" {
+					reason = " because: " + reason
+				}
```

Comment:
Let's go with the "because of an unknown reason" here to encourage people adding their the reasoning in!

Reply:
done - I've gone with "(no reason given)" as I think that makes more grammatical sense, and that the "(...)" should make it stand out a little more vs other stuff